tools/clang/test/DXC/embed_dbg.test

```diff
@@ -0,0 +1,13 @@
+// Embed debug info
+// RUN: %dxc /T ps_6_0 %S/Inputs/smoke.hlsl /Zi /Qembed_debug /Fo %t.embedpdb
+// RUN: %dxc -dumpbin %t.embedpdb | FileCheck  %s --check-prefix=EMBED_PDB
+
+
+// Auto-embed debug info when no debug output, and expect warning signifying that this is the case.
+// RUN: %dxc /T ps_6_0 %S/Inputs/smoke.hlsl  /Zi /Fo %t.embedpdb2 /Fe %t.err.embedpdb
+// RUN: FileCheck --input-file=%t.err.embedpdb %s --check-prefix=ERR_EMBED_PDB
+// ERR_EMBED_PDB:warning: no output provided for debug - embedding PDB in shader container.  Use -Qembed_debug to silence this warning.
```

Comment:
not cat *pdb works in this case.

Reply:
In conversation with Chris, I think perhaps `file -E` might be a better method as it doesn't have any issue with binary files and I expect it will be quicker